packages/twenty-front/src/modules/settings/data-model/objects/forms/components/SettingsDataModelObjectAboutForm.tsx

```diff
@@ -168,7 +171,9 @@ export const SettingsDataModelObjectAboutForm = ({
                 }
               }}
               onBlur={() => onNewDirtyField?.()}
-              disabled={disableEdition}
+              disabled={
+                !(objectMetadataItem?.isCustom ?? true) && isLabelSyncedWithName
```

Comment:
This is makes it hard to read. Not possible to have just `disabled={!objectMetadataItem?.isCustom && isLabelSyncedWithName}`? (didn't try)

Reply:
That was the consequence of multiple successive edits, should really have checked it this frightening expression wasn't really over complicated before pushing :sweat_smile: